adding scheduler cli-examples (#8983)
aws/aws-cli: awscli/examples/scheduler/tag-resource.rst

> **To assigns one or more tags (key-value pairs) to the specified EventBridge Scheduler resource**

Reviewer: nit: assigns -> assign

Author: updated

**To assigns one or more tags (key-value pairs) to the specified EventBridge Scheduler resource**

> The following ``tag-resource`` assigns one or more tags (key-value pairs) to the specified EventBridge Scheduler resource. If the command succeeds, no output is returned. ::

Reviewer 2: ```suggestion
The following ``tag-resource`` example assigns one or more tags (key-value pairs) to the specified EventBridge Scheduler resource. ::
```

Author: file fixed

	aws scheduler tag-resource \
		--resource-arn "arn:aws:scheduler:ca-central-1:123456789012:schedule-group/test" \

> 		--tags Key=environment,Value=production

Reviewer 2: ```suggestion
		--tags Key=environment,Value=production

This command produces no output.
```

Author: file fixed
